src/DataDictionary/DictionaryGraph.test.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import DictionaryGraph from './DictionaryGraph';
+import { buildTestData } from '../DataModelGraph/testData';
+
+
+describe('the DictionaryGraph', () => {
+  function buildTest() {
+    const data = buildTestData();
+    // Material-UI components require the Mui theme ...
+    const $dom = mount(
```

Comment:
use enzyme to find and select the DOM

Reply:
Hi Thanh!

That 'mount' function is an enzyme function - I was just naming my variables with a '$' sign to mark them as being enzype wrappers, but I can remove the '$' from the names if  that's to weird or confusing - what do you think?